data/incipias/incipias first contact.txt

```diff
@@ -0,0 +1,323 @@
+			`	"We shall allow you to buy one. If that is all that it takes to make you a not-danger. If that is all please excuse us, we need to talk about what this new situation means for us, I hope we can out-question you more another time."`
+				goto end
+			label stranded
+			`	Another wave goes through the crowd but this time you notice an overflow in green tones.`
+			`	"So they are still alive. We shall give you access to our ship. Let us prepare the ship-seller. Do notice though that they can not jump between home-places. We hope to question you more on another future-time.`
+			label end
+			`	The five float away but the crowd stays continues staring at you, probably out of interest.`
+
+mission "Incipias: Help the stranded Part 2"
+	landing
+	passengers 5
+	source
+		planet Turra
+	destination "Pon'tes"
+	to offer
+		has "Incipias: Help the stranded Part 1: done"
+	on offer
+		conversation
+			`Your ship dives into the clouds of the gas giant, an environment on a scale you never experienced before. Giant clouds tower like castles, sometimes opening to giant caverns. Whirlwinds brush through the atmosphere like pillars holding the sky. Even though the winds are brutal your ship glides through them as if they had no effect at all.`
+			`	You activate your scanners looking for the stranded ship. Soon a small blip appears on your screen, and you maneuver your ship to that point. The stranded ship looks even worse than the first time; the storms seem to have damaged some of the hull, and the shields must have broke when they ran out of fuel.`
+			`	You receive no answer to your hail. Your scanners indicate that it ran out of energy.`
+			`	You carefully maneuver your ship closer, until you can dock with the Incipias ship.`
+			`	When the lock opens five Incipias float in front of you. Noticing your presence they quickly back off; they were clearly not expecting a human in one of their kind's ships. It takes a moment for them to recognize you.`
+			choice
+				`	"Hello, I have come to save you."`
+				`	"I have been sent by your people to get you home."`
+			`	They seem to switch from a tense state to a surprised one.`
+			`	One of them starts pulsating, "You light our language? How surprising. We would not have survived anothey day here. You are our life-protector. We would love to talk more, but a weather-calamity is coming, we should leave now."`
+			`	You tell them to come aboard and launch away from the planet.`
+				accept
+	on complete
+		conversation
+			`When you land on the station your passengers thank you again and leave, probably to get some well-earned rest.`
+			`	As you step out of the <ship> an Incipias approaches you. "Follow me outer-helper before a crowd gathers, we would like to talk."`
+			`	They guide you through the hallways so that you avoid larger groups and move only through horizontal corridors. You arrive in a large room with a view over the entire station. Three Incipias are already in the room. One starts glowing.`
```

Comment:
```suggestion
			`	They guide you through the hallways so that you avoid larger groups and move only through horizontal corridors. You arrive in a large room with a view over the entire station, with three Incipias already present. One begins to glow.`
```